davidia/setup.py

```diff
@@ -1,13 +1,17 @@
+import os
 from setuptools import find_packages, setup
 
 # from Davidia import __version__
 __version__ = "0.1.0"
 
-
 setup(
     name="Davidia",
     version=__version__,
     description="Plot server with React frontend data visualiser",
+    long_description=open(
```

Comment:
You may have to be smarter than this as the build and rebuild happens in different directories. Perhaps, add function to find README by checking in grandparent directory and then this directory. Also include it with
`package_data={'': [ readme_file ]}`

Reply:
Or use a `MANIFEST.in` file